cpp/build.sh

```diff
@@ -1,4 +1,6 @@
 rm -rf build
 cmake CMakeLists.txt -B build
 cmake --build build --config Release
-cp build/lib/libjxtp.so ../lib
+lib_ext=".so"
+lib_ext=$([[ $(uname) = "Darwin" ]] && echo ".dylib")
```

Comment:
The lib_ext will be empty under Linux. Did I miss something? The following approach works fine for me:
```suggestion
if [[ "$(uname -s)" == "Darwin" ]] ; then
  lib_ext=".dylib"
fi
```